inc/postHead.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * The template for displaying a post header.
+ *
+ * @package MIT_Libraries_Child
+ * @since 2.0.0
```

Comment:
This isn't the Child theme anymore so this should get updated.

Reply:
ok